File: server/src/main/java/com/sju18/petmanagement/domain/community/dao/Post.java
package com.sju18.petmanagement.domain.community.dao;

import com.sju18.petmanagement.domain.account.dao.Account;
import com.sju18.petmanagement.domain.pet.dao.Pet;
import lombok.*;

import javax.persistence.*;
import java.time.LocalDateTime;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Post {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "post_id")
    private Long id;

    @ManyToOne(targetEntity = Account.class, fetch = FetchType.EAGER)
    @JoinColumn(name = "account_id")
    private Account author;

    @ManyToOne(targetEntity = Pet.class, fetch = FetchType.EAGER)
    @JoinColumn(name = "pet_id")
    private Pet pet;

    @Column(nullable = false)
    private String contents;
    @Column(nullable = false)
    private LocalDateTime timestamp;
    @Column(nullable = false)
    private Boolean edited;

    @Column(name = "tag_list")
    private String serializedHashTags;

    @Column
    private String disclosure;
    private Double geoTagLat;
    private Double geoTagLong;
    private String attachedImagesJsonArray;
}

File: client/android/app/src/main/java/com/sju18001/petmanagement/restapi/dao/Post.kt
package com.sju18001.petmanagement.restapi.dao

import java.time.LocalDateTime

data class Post(
    val id: Long,
    val author: Account,
    val pet: Pet,
    val contents: String,
    val timestamp: LocalDateTime,
    val edited: Boolean,
    val serializedHashTags: String,
    val disclosure: String,
    val geoTagLat: Double,
    val geoTagLong: Double,
    val mediaAttachments: String,
    val fileAttachments: String
)